ruby-marshal.cabal

```diff
@@ -29,12 +29,12 @@ library
     src
   build-depends:
       base        >= 4.7    && <= 5
-    , cereal      >= 0.4.0  && <  0.5.6
+    , cereal      >= 0.4.0  && <  0.5.9
     , bytestring  >= 0.9.0  && <= 0.12.0
-    , containers  >= 0.5.0  && <= 0.6.0
+    , containers  >= 0.5.0  && <  0.6.6
     , string-conv >= 0.1    && <= 0.2
     , mtl         >= 2.1.0  && <= 2.3.0
-    , vector      >= 0.10.0 && <  0.12.1
+    , vector      >= 0.10.0 && <  0.12.4
```

Comment:
I wonder why such specific bounds are used. I think `< 0.13` should suffice here. (0.13 would be the next "major" version)

Reply:
Yes, what I find unclear is the discrepancy between https://pvp.haskell.org/ and versioning used in many hackage packages that use a.b.c.d scheme. Is b or c the major version in these cases?